docs/src/routes/_layout.svelte

```diff
@@ -27,7 +27,6 @@
 
 <style global lang="less">
   @import "../theme/index.less";
-  @import "../../src/style/index.less";
```

Comment:
We need to add this back in, removing this will remove some general component styling.